doc/production.rst

```diff
@@ -74,3 +83,51 @@ tag you want to install on the machines:
    argument from both the ``discos-deploy`` and ``discos-get`` scripts. If you
    pass the ``--station`` argument anyway, if the given argument does not match
    the correct station you will receive an error and the procedure will stop.
+
+Replace the Manager in case of failure
+--------------------------------------
+In case the Manager machine suffers a failure of some sort, it has to be
+replaced. In order to do this, the first thing to do is perform again the
+provisioning procedure on a newly installed machine (after putting the new
+Manager's IP address in the Ansible inventory's hosts file). In order for
+the whole system to behave correctly it is also necessary to perform some
+manual tweaking on the other DISCOS machines as well (in case the DISCOS
+control system is running on a distributed environment. This is the case
+for the SRT and Medicina stations).
+
+The tweaks to be performed in order for the DISCOS control system to work as
+expected are the following:
+
+- Replace the old ACS Manager IP address reference with the new one in
+  ``/discos-sw/config/misc/bash_profile`` file in the ``discos-console``
+  machine. It is stored as an environment variable called ``MNG_IP``.
+- Replace the old Manager IP address with the new one in some fiels in the
+  DISCOS CDB. More specifically, one file has to be corrected in order for the
+  control system to be able to properly communicate with the ``TotalPower``
+  backend, you can find this file in the repository of the currently deployed
+  released of DISCOS, under the directory
+  ``SRT/Configuration/CDB/alma/BACKENDS/TotalPower/TotalPower.xml``.
+  The variable to be corrected is called ``DataIPAddress``. This has to be
+  performed on the new Manager machine itself before launching the control
+  system.
+- Make sure that all the station systems and machines accept incoming
+  connections from the newly allocated Manager's IP address. Specifically, the
+  ``TotalPower`` backend and the ``CalMux`` machines have to be tweaked in
+  order to allow them to be controlled by the new manager.
```

Comment:
Where is the procedure?

Reply:
This procedure involves logging in the said machines as root, if it has to be documented, this is not the place to do it. A suggestion about this is we perform this step in advance by allowing a range of addresses to control the said machines, so, in case of failure, this step can be skipped.

Reply:
No clear to me how it is possible to replicate the manager without any information about this point. I think the procedure should be documented somewhere, and in case this is not the place, here we have to put a reference link to it.